app/scripts/athena_partition_archive.py

```diff
@@ -65,6 +65,90 @@
 ATHENA = boto3.client('athena', config=CONFIG)
 GLUE = boto3.client('glue', config=CONFIG)
 
+# Below functions have been added  as part of improvements to the
+# maintenance script so it uses glue API's to drop and create partitions.
+
+def create_partition(partlist, database_name, table_name):
```

Comment:
Variable name in function: `partlist`
Variable name in comment: `partition_val`

Reply:
Corrected.